modules/tinymce/src/core/test/ts/browser/bookmark/BookmarksTest.ts

```diff
@@ -223,4 +223,50 @@ describe('browser.tinymce.core.bookmark.BookmarksTest', () => {
     assertApproxRawContent(editor, '<p>abc</p>');
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
   }));
+
+  context('Get bookmark should work if selection is on a comment', () => {
+    const hook = TinyHooks.bddSetupLight<Editor>({
+      base_url: '/project/tinymce/js/tinymce'
+    }, [], false);
+
+    const outsideButton: SugarElement<HTMLButtonElement> = SugarElement.fromHtml('<button id="getBookmarkButton">Get Bookmark</button>');
+    const setSelectionToCommentAndTriggerGetBookmark = (editor: Editor, content: string) => {
+      editor.resetContent(content);
+      editor.addCommand('getBookmarkProxyCommand', () => {
+        editor.selection.getBookmark();
+      });
+
+      const clickHandler = (): void => {
+        editor.execCommand('getBookmarkProxyCommand');
+      };
+      outsideButton.dom.addEventListener('click', clickHandler);
+
+      outsideButton.dom.click();
+      outsideButton.dom.removeEventListener('click', clickHandler);
```

Comment:
These should be using the relevant Sugar/Agar APIs instead, so in this case it would be something like this:
```ts
const unbind = DomEvent.bind(outsideButton, 'click', () => {
  editor.execCommand('getBookmarkProxyCommand');
});
Mouse.click(outsideButton);
unbind();
```